docs/docs/sr.md

```diff
@@ -0,0 +1,72 @@
+# Quantum Geometric Tensor ( and Stochastic Reconfiguration )
+
+The Quantum Geometric Tensor (QGT) is the Fubini-Study metric tensor of the manifold on which a variational state is defined.
+In general it is not possible to compute a form of the QGT valid on the whole manifold, but we can compute local approximations valid in small region around a given point $ W $
```

Comment:
I think this is mathematically imprecise? The metric on a manifold is always defined locally (of course it can happen to be the same at each point in a flat geometry), so making it sound like we are approximating something (a "exact" QGT that would be valid on the whole manifold) is a bit misleading since such an object just doesn't exists (so there is nothing to "approximate"). We _are_ computing the QGT at the specific point _W_; the approximation only happens when we do VMC.
```suggestion
In the general case (and in practise for any NQS ansatz), the QGT varies depending on the current quantum state and thus needs to be computed when the variational parameters $W$ change.
```

Reply:
Yes you are right.
However we are still computing the linearization of the metric tensor on a point W , not the actual metric tensor, which is an approximation.

Reply:
Sure? I think the [metric tensor](https://en.wikipedia.org/wiki/Metric_tensor) on a manifold includes this linearization by definition. Note how the Wikipedia article says:
> Equipped with this notion of length [= length of a geodesic between two points], a Riemannian manifold is a metric space, meaning that it has a distance function d(p, q) whose value at a pair of points p and q is the distance from p to q. Conversely, the metric tensor itself is the derivative of the distance function (taken in a suitable manner). Thus the metric tensor gives the infinitesimal distance on the manifold.

So the metric tensor already gives the infinitesimal distance around a given points, which is what you are saying, right? But in that sense it's really not an approximation. It does locally approximate the _metric_ (in the sense of distance function), but that's a different object.

(Just to be clear, I'm not arguing this to be pedantic - we already agree on the formulation in the docs I think. I'd just like to understand whether we use the same terminology or not. :slightly_smiling_face: )